server/utils/config.ts

```
@@ -298,9 +298,9 @@ export class ConfigService {
 
   private setSSOParams = (config) => {
     if (process?.env?.RTL_SSO) {
-      config.SSO.rtlSso = +process?.env?.RTL_SSO;
+      config.SSO.rtlSSO = +process?.env?.RTL_SSO;
     } else if (config.SSO && config.SSO.rtlSSO) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)